_variables.tf

```diff
@@ -30,3 +30,49 @@ variable "project" {
   type    = string
   default = null
 }
+
+variable "shared" {
+  type        = bool
+  default     = false
+  description = "Set to true if this topic ownership is shared between differents teams, i.e. different teams publish to it"
+}
+
+variable "team" {
+  type        = string
+  default     = null
+  description = "The team that owns the topic"
+
+  validation {
+    condition = contains([
+      "analytics",
+      "devops",
+      "label-platform",
+      "leadflow",
+      "office",
+      "scraper"
+    ], var.team)
+    error_message = "Unrecognized team, the following teams are allowed: ${join(", ", [
+      "analytics",
+      "devops",
+      "label-platform",
+      "leadflow",
+      "office",
+      "scraper",
+    ])}"
```

Comment:
Would love to have these come out of a central data module, but we can't have nice things with how limited terraform is